appliance-root/opt/noit/prod/etc/circonus-modules-enterprise.conf

```diff
@@ -10,7 +10,6 @@
 <module image="httptrap" name="httptrap">
   <config>
     <surrogate>true</surrogate>
-    <fanout>off</fanout>
```

Comment:
Is this on by default? Should we explicitly specify "on"?

Reply:
It defaults to true. See https://github.com/circonus-labs/reconnoiter/blob/master/src/modules/httptrap.c#L1250